src/main/java/frc/robot/OI.java

```diff
@@ -46,6 +47,10 @@ public static JoystickButton getOrientationButton(){
         public static JoystickButton getZeroButton(){
             return new JoystickButton(kJoystick, kZeroButtonID);
         }
+
+        public static JoystickButton getOneButton(){
+            return new JoystickButton(kJoystick, kOneButtonID);
+        }
```

Comment:
getZeroButton is the button that is meant to "Zero" the gyroscope heading.

We shouldn't use getOneButton, instead make the button descriptive of what the button is doing.

Reply:
Switched to getBalanceButton and button ID changed to kToggleBalanceButton